qvm/whonix.jinja

```diff
@@ -0,0 +1 @@
+{% set whonix_version = salt['pillar.get']('qvm:whonix:version', '14') %}
```

Comment:
Should we call the variable `whonix_version`?

Can the user set it from command line? To install a newer Whonix version?

I envision some day there could a gui that shows "Whonix version".

What does it mean?

* Installed Whonix version?
* Whonix version to be installed?

Don't let my comment block. Just a small nit.

Reply:
> `qvm:whonix:version`

> Should we call the variable `whonix_version`?

Maybe, but my thinking was that this way we can have an orderly tree, with more whonix-related variables there.